source/skymaps/index.rst

```diff
@@ -84,7 +84,11 @@ Header Keywords
       lower and upper edges of each bin.  If there is a single element
       the column should be interpreted as the bin center.  For
       ``EBOUNDS`` or ``ENERGIES`` HDUs this keyword is optional.
-  
+
+For time based axes the additional keywords described in `time-formats`_ are
+allowed.
```

Comment:
Change to required? I think if there is a time axis, we need the time header keywords, right?

Reply:
Yes, I agree. If there is a time axis it should specify absolute times. The previous specification allowed for a time axis with relative time without reference time specification, which is questionable.